src/snowflake/connector/aio/_network.py

```diff
@@ -82,13 +85,16 @@
     SQLSTATE_CONNECTION_WAS_NOT_ESTABLISHED,
 )
 from ..time_util import TimeoutBackoffCtx
+from ._description import CLIENT_NAME
 from ._ssl_connector import SnowflakeSSLConnector
 
 if TYPE_CHECKING:
     from snowflake.connector.aio import SnowflakeConnection
 
 logger = logging.getLogger(__name__)
 
+PYTHON_CONNECTOR_USER_AGENT = f"{CLIENT_NAME}/{SNOWFLAKE_CONNECTOR_VERSION} ({PLATFORM}) {IMPLEMENTATION}/{PYTHON_VERSION}"
```

Comment:
I'm thinking of reverting this change first because I'm not sure how user agent could affect the backend behavior.
we need to understand what user agent is used in the backend first